superset/models/helpers.py

```diff
@@ -544,13 +545,28 @@ def changed_on_utc(self) -> str:
         # Convert naive datetime to UTC
         return self.changed_on.astimezone(pytz.utc).strftime("%Y-%m-%dT%H:%M:%S.%f%z")
 
+    def _format_time_humanized(self, timestamp: datetime) -> str:
```

Comment:
Seems this improvement should be backend only (?) I'm wondering why we need to touch the frontend at all.